apps/cli/src/node/docker-compose-dev.yaml

```diff
@@ -159,14 +174,26 @@ services:
             validator:
                 condition: service_healthy
         healthcheck:
-            test: ["CMD", "traefik", "healthcheck"]
+            test: ["CMD", "traefik", "healthcheck", "--ping"]
             interval: 5s
             timeout: 1s
             retries: 5
+        command:
+            [
+                "--ping=true",
+                "--entryPoints.web.address=:8088",
+                "--entryPoints.traefik.address=:8080",
+                "--metrics.prometheus=true",
+                "--metrics.prometheus.addServicesLabels=true",
+                "--providers.docker.exposedByDefault=false",
+                "--providers.docker=true",
+                "--providers.docker.endpoint=unix:///var/run/docker.sock",
+            ]
         ports:
-            - 8080:8080
+            - 8080:8088
+            - 8088:8080
         volumes:
-            - ${SUNODO_BIN_PATH}/node/traefik:/etc/traefik
+            - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Yeah, this is not a good option. We may have problems with different docker installations, and privileged vs unprivileged modes.

Reply:
I think we should go the CLI arguments path, and avoid exposing the `docker.sock`, since this would be listening to any other container events running on the developer machine.

Reply:
> I think we should go the CLI arguments path, and avoid exposing the `docker.sock`, since this would be listening to any other container events running on the developer machine.

We can use `[onstraints](https://doc.traefik.io/traefik/providers/docker/#constraints) to avoid this. 

But, the main question is: Are there any solutions that don't need to mount the `docker.sock` volume  and config files?

Reply:
Yes, CLI arguments or ENV variables.

https://doc.traefik.io/traefik/reference/static-configuration/cli/

https://doc.traefik.io/traefik/reference/static-configuration/env/

Reply:
@endersonmaia There is no option to define all routes and services throw args nor ENVs. 

We should use one the traefik providers at first (we can define it via ENV or CMD). then we should configure the routes in the provider section. So, based on our scenario we have two options : 

1 - docker provider (we just need to mount docker.sock)
2 - file provider (we need to mount the config files --> yaml or toml format)


So, I think we can use docker format to remove the files, but one problem is the traefik should connect to docker.sock to read the labels + discover the IP of the containers.

Reply:
So the premises are:

-  routers, services and middleware cant be hold in the static configuration, but only via providers
- file provider is the solution to use, but we don't want to mount a local volume

While talking to Omid about this, we thought of creating a new service just to provision this configuration on a docker volume for the `proxy` service to use.

The static part could be on cli arguments, like this:

```
    command
          - --entrypoints.web.address=:8080
          - --entrypoints.traefik.address=:8088
          - --ping
          - --providers.file.directory=/etc/traefik/conf.d
          - --metrics.prometheus.addServicesLabels=true
```

And we manage to feed this `/etc/traefik//conf.d` directory.

This way we avoid mounting local host's volume and also avoid maintaining another container image.

Reply:
![image](https://github.com/sunodo/sunodo/assets/51019382/beea8421-ef00-45b0-8cac-fb6f190c5442)


![image](https://github.com/sunodo/sunodo/assets/51019382/5925bfb0-43dd-47ae-bb9f-f09f143a6232)


![image](https://github.com/sunodo/sunodo/assets/51019382/648e8702-5de2-4db7-b917-f042559eeb59)


![image](https://github.com/sunodo/sunodo/assets/51019382/8112cb56-04b1-4c99-b6f6-7c8a861351ed)